#Numpy raises an error, because it does not find our X11 libs and headers because
#they are installed at non-standard location.
# notfound_action=2 raises an error, changing to notfound_action=1 issues only a warning/
# August 17th 2016 by B. Hajgato (Free University Brussels - VUB)
--- enable-b8eea9cc642388cd331452f95dd19dd0672d2241/kiva/agg/setup.py.org	2016-08-17 10:09:54.000000000 +0200
+++ enable-b8eea9cc642388cd331452f95dd19dd0672d2241/kiva/agg/setup.py	2016-08-17 10:40:02.606906718 +0200
@@ -235,7 +235,7 @@
         # Frequently, the 64-bit libraries are not in a known location and need
         # manual configuration. From experience, this is usually not detected by
         # the builder if we do not raise an exception.
-        x11_info = get_info('x11', notfound_action=2)
+        x11_info = get_info('x11', notfound_action=1)
         dict_append(plat_info, **x11_info)
 
     elif plat=='gdkpixbuf2':
